service/controller/resource/containerurl/error.go

```diff
@@ -32,7 +32,8 @@ func IsStorageAccountNotFound(err error) bool {
 	if err == nil {
 		return false
 	}
-	return strings.Contains(microerror.Cause(err).Error(), "ResourceNotFound")
+	return strings.Contains(microerror.Cause(err).Error(), "ResourceNotFound") ||
```

Comment:
Apparently the Azure response for the 404 case changed from a generic one to a specific one recently.